lib/internal/test_runner/mock/mock.js

```diff
@@ -2,6 +2,11 @@
 const {
   ArrayPrototypePush,
   ArrayPrototypeSlice,
+  ArrayPrototypeShift,
+  ArrayPrototypeAt,
+  SetPrototypeAdd,
+  SetPrototypeClear,
+  SetPrototypeDelete,
```

Comment:
Since the `Set` is created with `SafeSet` we don't really need these 🙂

Reply:
thanks, done